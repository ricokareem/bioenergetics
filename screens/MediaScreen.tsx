import React, { ReactElement } from "react";
import { Route } from "react-native";
import { OrangeContainer } from "../components";
import MediaCard from "../components/MediaCard";

type MediaScreenProps = {
  route: Route;
};

function MediaScreen({ route }: MediaScreenProps): ReactElement {
  const { title, playlist, showTimer } = route.params;

  return (
    <OrangeContainer>
      <MediaCard title={title} playlist={playlist} showTimer={showTimer} />
    </OrangeContainer>
  );
}

export default MediaScreen;
