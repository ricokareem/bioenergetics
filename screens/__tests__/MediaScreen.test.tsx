import React from "react";
import { shallow } from "enzyme";
import MediaScreen from "../MediaScreen";

describe("<MediaScreen>", () => {
  it("should match to snapshot", () => {
    const wrapper = shallow(
      <MediaScreen
        route={{ params: { title: "Media Screen", playlist: [81, 82] } }}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
