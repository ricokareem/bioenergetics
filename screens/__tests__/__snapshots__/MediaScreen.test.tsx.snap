// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaScreen> should match to snapshot 1`] = `ShallowWrapper {}`;
